Makefile

```diff
@@ -51,6 +52,7 @@ spell: ## misspell
 .PHONY: lint
 lint: ## golangci-lint
 	$(call print-target)
+	yamllint .
```

Comment:
I suggest splitting `lint` to two targets: `golint` and `yamllint`. We can still have `lint` which would depend on both of them.